pkg/service/rtcservice.go

```diff
@@ -517,7 +517,7 @@ type connectionResult struct {
 func (s *RTCService) startConnection(ctx context.Context, roomName livekit.RoomName, pi routing.ParticipantInit, timeout time.Duration) (connectionResult, *livekit.SignalResponse, error) {
 	var cr connectionResult
 	var err error
-	cr.Room, err = s.roomAllocator.CreateRoom(ctx, &livekit.CreateRoomRequest{Name: string(roomName)})
+	cr.Room, _, err = s.roomAllocator.CreateRoom(ctx, &livekit.CreateRoomRequest{Name: string(roomName)})
```

Comment:
What happens if that Room is created implicitly because someone entered the room? shouldn't we start an egress as well?

Reply:
right now it's only on explicit `CreateRoomRequest`